### DISCUSSION TITLE
Create Node on Output button

### DISCUSSION THREAD

**Discussion by gielians** [VentuzTechnology/Ventuz.Extension.prerelease#7] [Q&A]
Hi,
I made my first Ventuz node (yay) and did not have any issues (see JsonQuery example).
It took me about one hour which I think is pretty impressive, given the fact that it is the first time nad I had to set up everything.
So great work so far. Other things I have in mind:
- understand to ressource handling better and play around with it.
- build some geometry things like a Bar Chart in one node...
Unfortunately I don't have much time right now, so we'll see when I can play around any further...

My Question:
is there a way to get the button at the Output like on the JsonParser Node which creates a JsonParser Node automatically? The JsonParser output type is `JsonObject` whis is not included in the assemblies I loaded...

**Comment by VentuzDanielWiller** [comment]
Hey Eric,

I cannot answer your question but congrats to your first Ventuz Extension. An hour is quite impressive.
@VentuzRalfStanke  and @VentuzBjoernGdanietz can hopefully offer more than just applaus

**Comment by VentuzRalfStanke** [comment]
Hi Eric, creating nodes is part of the "tree-builder-API" which has not been opened yet (V0.9). But yes, as soon this API is there you can create node (e.g. by "custom verb") and bind them to others.
